docs/make-docs

```diff
@@ -478,6 +490,8 @@ POSIX_HERESTRING
 
   echo
   errr 'The build was interrupted or a build error occurred, check the previous logs for possible causes.'
+  note 'You might need to use Ctrl+c to end the process.'
```

Comment:
```suggestion
  note 'You might need to use Ctrl+C to end the process.'
```
For keys, use WYSIWYG. If we were to use shift, we would write Ctrl-Shift-C.

Reply:
Makes sense, I can make this change upstream and it will get propagated to all other repositories.